hedera-node/src/main/java/com/hedera/services/store/TypedTokenStore.java

```diff
@@ -391,6 +466,8 @@ private void initModelFields(Token token, MerkleToken immutableToken) {
 		token.setLastUsedSerialNumber(immutableToken.getLastUsedSerialNumber());
 		token.setIsDeleted(immutableToken.isDeleted());
 		token.setExpiry(immutableToken.expiry());
+
+		immutableToken.adminKey().ifPresent(token::setAdminKey);
```

Comment:
nit 1: lets keep it consistent and always set the adminKey similar to how we do with the rest of the keys
nit 2: Lets place the admin key along with the rest of the keys